Counter/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Counter</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header></header>
+    <main>
+      <div class="counter_background">
+        <div class="counter">
+          <button type="button" onClick="dec3()">
+            <a>-3</a>
+          </button>
+
+          <button type="button" onClick="dec2()">
+            <a>-2</a>
+          </button>
+
+          <button type="button" onClick="dec1()">
+            <a>-1</a>
+          </button>
+
+          <div class="number"><a id="num">0</a></div>
+
+          <button type="button" onClick="inc1()">
```

Comment:
```suggestion
          <button type="button" onClick="inc(1)">
```
Now instead of calling a separate function, you call a universal inc function with an attribute.